pkg/nmstate/policy.go

```diff
@@ -264,6 +264,73 @@ func (builder *PolicyBuilder) WithBondInterface(slavePorts []string, bondName, m
 	return builder.withInterface(newInterface)
 }
 
+// WithVlanInterface adds VLAN interface configuration to the NodeNetworkConfigurationPolicy.
+func (builder *PolicyBuilder) WithVlanInterface(baseInterface string, vlanID uint16) *PolicyBuilder {
+	if valid, err := builder.validate(); !valid {
+		builder.errorMsg = err.Error()
+
+		return builder
+	}
+
+	glog.V(100).Infof("Creating NodeNetworkConfigurationPolicy %s with VLAN interface configuration: %s %d",
```

Comment:
Do you have an example output of this message. It seems like it should be:

Creating NodeNetworkConfigurationPolicy %s with VLAN interface %s and vlanID %d

Reply:
changed